### PR TITLE
69 - Transition setup.py to pyproject.toml

### PR DESCRIPTION
**Pull request by mxochicale**
# Addressing #69

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Pycharm
 /.idea
 
+# Default project slug path
+**/MyNewProject
```
Comment (thompson318) [inline .gitignore:5]: I don't understand this addition. Why do we ignore MyNewProject?

**Review by thompson318 — COMMENTED** [review]
I'm a little confused. I thought we were moving away from tox. Historically we've used tox to install the requirements defined in requirements-dev.txt, so if tox is added to them what are we going to use to install?

MX: As discussed. I have added few packages to requirements-dev.txt for developing the package. Thanks, --Miguel

**Review by thompson318 — APPROVED** [review]
OK, I get it now.

File: .gitignore (hunk 1 above)
Comment (mxochicale) [reply .gitignore:5]: As discussed, I am adding few lines to ignore development files and paths. Thanks, --Miguel

**Comment by mxochicale** [comment]
Hi @thompson318 , this PR is ready for review. Let me know if you have further comments, otherwise it should be ready to be merged. Thanks, --Miguel

**Review by thompson318 — APPROVED** [review]

**Pull request merged**